src/Microsoft.Sbom.Api/Executors/ConcurrentHashValidator.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Sbom.Api.Entities;
+using Microsoft.Sbom.Api.Manifest.FileHashes;
+using Microsoft.Sbom.Contracts.Enums;
+using Microsoft.Sbom.Extensions.Entities;
+using System;
+using System.Linq;
+using System.Threading.Channels;
+using System.Threading.Tasks;
+
+namespace Microsoft.Sbom.Api.Executors
+{
+    /// <summary>
+    /// Validates hashes from multiple file locations (on disk and inside SBOM) simulatenously using
+    /// a conncurrent dictionary.
+    /// </summary>
+    public class ConcurrentHashValidator
```

Comment:
>ConcurrentHashValidator [](http://example.com/codeflow?start=17&length=23)

Should this be ConcurrentHash256Validator?

Reply:
fixed